src/field_control_avatar.c

```diff
@@ -154,6 +161,7 @@ int ProcessPlayerFieldInput(struct FieldInput *input)
     gSpecialVar_LastTalked = 0;
     gSelectedObjectEvent = 0;
 
+	gMsgIsSignPost = FALSE;
```

Comment:
spaces instead of tabs

Reply:
[`f0bd3c5` (#5044)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/f0bd3c57daa2a32eca39e01b8f10c92d85d8e4bf)